ocpmodels/trainers/forces_trainer.py

```diff
@@ -181,20 +167,19 @@ def load_task(self):
             self.relax_dataset = registry.get_dataset_class(
                 "single_point_lmdb"
             )(self.config["task"]["relax_dataset"])
-
-            self.relax_sampler = DistributedSampler(
+            self.relax_sampler = self.get_sampler(
                 self.relax_dataset,
-                num_replicas=distutils.get_world_size(),
-                rank=distutils.get_rank(),
+                self.config["optim"].get(
+                    "eval_batch_size", self.config["optim"]["batch_size"]
+                ),
                 shuffle=False,
             )
-            self.relax_loader = DataLoader(
+            self.relax_loader = self.get_dataloader(
                 self.relax_dataset,
-                batch_size=self.config["optim"].get("eval_batch_size", 64),
-                collate_fn=self.parallel_collater,
-                num_workers=self.config["optim"]["num_workers"],
-                pin_memory=True,
-                sampler=self.relax_sampler,
+                self.relax_sampler,
+                self.config["optim"].get(
+                    "eval_batch_size", self.config["optim"]["batch_size"]
```

Comment:
`batch_size` is not an argument to `get_dataloader`, has already been fed into the sampler, remove this line.

Reply:
Yes, that's a leftover from a small refactoring I did. Fixed in a6030eb.